xml/System.Collections.Immutable/ImmutableDictionary.xml

```diff
@@ -185,7 +185,7 @@
         <typeparam name="TKey">The type of keys stored by the dictionary.</typeparam>
         <typeparam name="TValue">The type of values stored by the dictionary.</typeparam>
         <param name="keyComparer">The implementation to use to determine the equality of keys in the dictionary.</param>
-        <param name="valueComparer">The implementation to use to determine the equality of values in the dictionary.</param>
+        <param name="valueComparer">The implementation to use to determine the equality of values in the dictionary. Replacing the value with an equal value does not mutate the collection.</param>
```

Comment:
We generally try to keep `param` documentation brief. I'd probably omit this and add a remark in the `ValueComparer` property, something along the lines of

> Equal values do not get replaced in order to minimize the amount of copying required when updating an immutable collection.